variables.tf

```diff
@@ -199,6 +199,18 @@ variable "primary_metastore_port" {
   default     = "9083"
 }
 
+variable "primary_metastore_host_read_only" {
```

Comment:
You should update the `README.md` and include them there as well.

Reply:
done